### DISCUSSION TITLE
cli: auth command should use admin RPC

### DISCUSSION THREAD

**Discussion by Wondertan** [celestiaorg/celestia-node#3030] [Ideas]
[discussion]
This should become a call on `Admin.AuthNew`. Likely in a separate PR

_Originally posted by @Wondertan in https://github.com/celestiaorg/celestia-node/pull/2762#discussion_r1339986003_

**Comment by vgonkivs** [comment]
After some work on this issue, I've realized that we should leave auth cmd as it is, bc in order to create a client, we already have to have an auth token. I've tested several use cases(run a LN and create a token for the LN - it works, run a LN - create a token for the FN - it also works, but that is incorrect). We also have a `set-permission ` cmd that does completely the same as your suggested version of `AuthCmd()`. So, we discussed it with @renaynay internally and decided to leave this particular cmd as it is.

**Comment by ramin** [comment]
it sounds like the decision has been made to NOT proceed with this? @vgonkivs can you confirm and close

**Comment by Wondertan** [comment]
>  bc in order to create a client, we already have to have an auth token.

That's totally fine as we always have the main key generated in KeyStore, so we always have a key to use and that's by design. 

I don't think we should close this yet.
